<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageLvs</class>
 <widget class="QWidget" name="PageLvs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>181</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>169</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>测试时间(s):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>功率上限(W):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="BoxTime">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>0.500000000000000</double>
        </property>
        <property name="maximum">
         <double>999.899999999999977</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>电流下限(A):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="BoxCurrMin">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>功率下限(W):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="BoxPowerMin">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>3000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>电流上限(A):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="BoxCurrMax">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="maximum">
         <double>5.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="BoxPowerMax">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="maximum">
         <double>3000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="BtnExit">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>44</height>
         </size>
        </property>
        <property name="text">
         <string>保存退出</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>频率(Hz):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>外部电压(V)：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="BoxVolt">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>255.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>220.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>通道:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="BoxPower">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>1.000000000000000</double>
        </property>
        <property name="maximum">
         <double>3.000000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="BoxFreq">
        <property name="minimumSize">
         <size>
          <width>97</width>
          <height>35</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>0</number>
        </property>
        <property name="minimum">
         <double>49.000000000000000</double>
        </property>
        <property name="maximum">
         <double>99.000000000000000</double>
        </property>
        <property name="value">
         <double>50.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>169</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>180</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
